sdk/communication/communication-administration/README.md

```diff
@@ -17,17 +19,31 @@ npm install @azure/communication-administration
 
 ## Key concepts
 
-### CommunicationIdentityClient
+### Clients
+
+The administration package exposes two (2) clients. The `CommunicationIdentityClient` provides methods to manage users and their tokens. The `PhoneNumberAdministrationClient` provides methods to manage phone plans and numbers.
+
+### Phone plans overview
+
+Phone plans come in two types; Geographic and Toll-Free. Geographic phone plans are phone plans associated with a location, whose phone numbers' area codes are associated with the area code of a geographic location. Toll-Free phone plans are phone plans not associated location. For example, in the US, toll-free numbers can come with area codes such as 800 or 888.
+
+All geographic phone plans within the same country are grouped into a phone plan group with a Geographic phone number type. All Toll-Free phone plans within the same country are grouped into a phone plan group.
 
-`CommunicationIdentityClient` provides methods to manage users and their tokens.
+### Searching and Acquiring numbers
+
+Phone numbers can be search through the search creation API by providing a phone plan id, an area code and quantity of phone numbers. The provided quantity of phone numbers will be reserved for ten minutes. This search of phone numbers can either be cancelled or purchased. If the search is cancelled, then the phone numbers will become available to others. If the search is purchased, then the phone numbers are acquired for the Azure resources.
+
+### Configuring and Assigning numbers
```

Comment:
```suggestion
### Configuring and assigning numbers
```

Also see Microsoft's writing style guide regarding capitalization here (it's UWP but it's the same across products):
https://docs.microsoft.com/en-us/style-guide/capitalization#sentence-style-capitalization-in-titles-and-headings